bin/github-changelog.php

```diff
@@ -84,13 +84,29 @@ function create_draft_changelog( $title, $content ) {
     echo $response;
 }
 
+function get_changelog_tags( $github_labels ) {
+    $tags = [
+        '1784989' // mu-plugins
+    ];
+
+    foreach ( $github_labels as $label ) {
+        preg_match('/ChangelogID:\s*(\d+)/', $label['description'], $matches);
```

Comment:
ChangelogID is a bit confusing since it may imply the Changelog post ID. I think ChangelogTagID is more descriptive.
```suggestion
        preg_match('/ChangelogTagID:\s*(\d+)/', $label['description'], $matches);
```